analytics/no_op_tracker.go

```diff
@@ -0,0 +1,57 @@
+package analytics
+
+import (
+	"path"
+
+	models2 "github.com/bitrise-io/bitrise/models"
```

Comment:
I think this should be imported without an alias and `stepman/models` should be aliased to `stepmanModels`. This would be consistent with other files in the repo.